src/player_hardcoded_effects.cpp

```diff
@@ -713,10 +713,10 @@ void player::hardcoded_effects( effect &it )
         }
         if( dur > 4_hours ) {
             // 8 teleports
-            if( one_in( 10000 - to_turns<int>( dur ) ) && !has_effect( effect_valium ) ) {
+            if( one_in( 10000 - ( dur / 6_seconds ) ) && !has_effect( effect_valium ) ) {
```

Comment:
I'd rather not have anything that's like "previous value / 6", rather changing the new value to make sense with 1 second turns. i even believe there is a `once_in( time_duration )` function somewhere.

Reply:
I guess maybe `one_in( 10000 - ( dur - 240_minutes ) / 4_turns )` can be a choice? The previous changes is only meant to keep consistency because I was not sure whether a brand new value is allowed or not.

Reply:
`one_in( to_seconds<int>( 1000_minutes - dur ) )` seems the most like what it was originally.

Reply:
Ye...s, sorry I forgot about the judgment frequency.